subsys/esb/Kconfig

```
@@ -80,4 +80,12 @@ endchoice
 
 endmenu
 
+config ESB_DYNAMIC_INTERRUPTS
+	bool "Use direct dynamic interrupts for ESB IRQ handlers"
```

Maybe just keep it simple like:
```suggestion
	bool "Use direct dynamic interrupts"
```

---

Done